cmd/acraserver/acraserver.go

```diff
@@ -169,14 +178,23 @@ func main() {
 			Errorln("Can't initialise keystore")
 		os.Exit(1)
 	}
-	if *useTls {
-		log.Infof("Selecting transport: use TLS transport wrapper")
-		tlsConfig, err := network.NewTLSConfig(*tlsSNI, *tlsCA, *tlsKey, *tlsCert)
+	var tlsConfig *tls.Config
+	if *useTls || *tlsKey != "" {
+		tlsConfig, err = network.NewTLSConfig(*tlsSNI, *tlsCA, *tlsKey, *tlsCert)
 		if err != nil {
 			log.WithError(err).WithField(logging.FieldKeyEventCode, logging.EventCodeErrorTransportConfiguration).
 				Errorln("Configuration error: can't get config for TLS")
 			os.Exit(1)
 		}
+		// need for testing with mysql docker container that always generate new certificates
+		if Test == TEST_MODE {
```

Comment:
Won't it be better to rename `Test` into `SkipVerifyingCertificated`?

Reply:
thought to name it more common to use in future tests if need

Reply:
what about 'IsRunningTest'? 'TestOnly'?

Reply:
by the way, why do you check 
`if Test == TEST_MODE`

instead of `if Test == True`?

Reply:
on compile time we can change only string values

Reply:
will rename to TestOnly